goosebit/settings/schema.py

```diff
@@ -39,7 +39,7 @@ class MetricsSettings(BaseModel):
 
 
 class GooseBitSettings(BaseSettings):
-    model_config = SettingsConfigDict(env_prefix="goosebit_")
```

Comment:
Please squash such commits created by the UI (locally) .

Reply:
This easily be done as follows:

1. `git rebase -i origin/master`
2. replace `pick` with `f` at the appropriate line.
3. CLose the editor.